profiles/Santosh1176.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/91916466/169870632-0ab75c94-7045-4aee-be36-6ecd734d06b4.png",
+  "issueId": 666,
```

Comment:
I think this is the wrong issue number? 👀

Reply:
I’ve made necessary changes to #issue number. I hope it’s ok now.